### PR TITLE
Fixed yaml model (insecureSkipTlsVerify)

### PR DESCRIPTION
**Pull request by wprzytula**
Field insecureSkipTlsVerify was incorrectly put in AuthInfo instead of Datacenter.

### CODE REVIEW

**Comment by wprzytula** [comment]
@avelanarius

File: driver-core/src/main/java/com/datastax/driver/core/Cluster.java [hunk 1]
```diff
@@ -1408,7 +1408,7 @@ protected Builder withScyllaCloudConnectionConfig(ScyllaCloudConnectionConfig co
                     new ScyllaCloudSniEndPointFactory(
                         proxyAddress, currentDatacenter.getNodeDomain()))
                 .withSSL(
-                    (config.getCurrentAuthInfo().isInsecureSkipTlsVerify()
+                    (config.getCurrentDatacenter().isInsecureSkipTlsVerify()
                         ? config.createBundle().getInsecureSSLOptions()
                         : config.createBundle().getSSLOptions()))
```
Comment (avelanarius) [inline driver-core/src/main/java/com/datastax/driver/core/Cluster.java:1413]: A potential problem I see (and I didn't expect at the beginning): Previously, there was only a single `insecureSkipTlsVerify`, so we could set it here with `withSSL` on a session. But now, you could have different configurations of `insecureSkipTlsVerify` per each datacenter - but here `currentDatacenter` determines the configuration for all SSL connections.

**Comment by avelanarius** [comment]
@wprzytula Could you rebase on top of latest `scylla-3.x`?

I think that we will merge the current version (since Python version with the same problem was already merged and CCM was changed) and just put an issue to fix this in the future - the first iteration of serverless is a single-DC serverless.

**Comment by avelanarius** [comment]
@wprzytula Sorry to ask again for rebasing - the test failed due to unrelated reasons to this PR. The `scylla-3.x` now contains the fix that should fix it (just committed it: https://github.com/scylladb/java-driver/commit/b49aa5dd1c8e664efe2f948403d884578b4131dd).

**Comment by avelanarius** [comment]
@wprzytula Merged the commit. The commit message should be more descriptive of the change (not generic "fixed yaml model"). I fixed it myself - https://github.com/scylladb/java-driver/commit/22e3ad5dc0de4cea8fe45370ff6fa4650c4bafd1

**Pull request merged**